frontend/src/audios.html

```diff
@@ -8,8 +8,17 @@
   - topnav
 jsFile: audios.js
 ---
-<div class="mt-4 flex">
-  <div id="filter-panel" class="fixed top-0 z-20 hidden h-full w-64 max-w-full flex-col bg-white text-sm open:flex md:static md:z-auto md:flex md:w-1/5 md:max-w-sm">
+<div
+  x-data="{open: false}"
+  x-init="$watch('open', (value) => $dispatch('toggle-backdrop', value))"
+  class="mt-4 flex"
+>
```

Comment:
Here I believe we don't need to touch the outer div, which wraps the filter panel and the main block. This has a performance penalty. Alpine has to scan all the elements inside the `x-data` element, including all the hit results. This is too excessive.

Instead, make the filter panel a component, and the filter button another, data-less component. Then dispatch an event from the filter button to the filter panel to open it. That's it.

Reply:
code updated with filter panel and filter button made two components and filter panel opened with custom event.